CHANGELOG.md

```diff
@@ -1,3 +1,168 @@
+# 60.2.0
+
+## Breaking changes
+
+## New deprecations
+
+* Deprecated `MDCAppBarTextColorAccessibilityMutator`.
+
+## Will be deprecated
+
+### FlexibleHeader
+
+* [minMaxHeightIncludesSafeArea](https://github.com/material-components/material-components-ios/issues/4764)
+
+## New features
+
+* AlertController got a new buttonInkColor property.
+
+## API changes
+
+### AppBar
+
+#### MDCAppBarTextColorAccessibilityMutator
+
+*deprecated* class: `MDCAppBarTextColorAccessibilityMutator`
+
+*deprecated* method: `-mutate:` in `MDCAppBarTextColorAccessibilityMutator`
+
+### Dialogs
+
+#### MDCAlertController
+
+*new* property: `buttonInkColor` in `MDCAlertController`
+
+#### MDCAlertControllerView
+
+*new* property: `buttonInkColor` in `MDCAlertControllerView`
+
+### FlexibleHeader
+
+#### MDCFlexibleHeaderView()
+
+*moved* category: `MDCFlexibleHeaderView()`
+
+*modified* property: `contentView` in `MDCFlexibleHeaderView()`
+
+## Component changes
+
+### MaskedTransition
+
+#### Changes
+
+* [[List] Fix Swift imports (#4828)](https://github.com/material-components/material-components-ios/commit/b917bf24d60029d76bf29114a2c1526793dfe6b1) (Robert Moore)
+
+### FeatureHighlight
+
+#### Changes
```

Comment:
@jverkoey Why does every section have a "Changes" subsection? I don't see anything *except* changes in here.  This is already in a supersection titled, "Component changes."

Can we remove these `### Changes` lines since they are just adding clutter and making it harder to read?

Reply:
Because there can be breaking changes for a given component.

Reply:
And changes to the script are welcome - send a PR!

Reply:
And changes to the script are welcome - send a PR!